kafka/getmetrics_kafka.py

```diff
@@ -308,8 +317,8 @@ def parseConsumerMessages(consumer, grouping_map, all_metrics_set):
                         # update the collected metrics for this timestamp
                         collectedMetricsMap[epoch] = collectedMetricsSet
                 elif metric_class == "filesystem":
```

Comment:
we don't need to list filesystem specially here. It should just read metrics from config file